content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+* DNS A record and TCP/IP access to port 443 for the SPIRE OIDC [Discovery Provider REST API](/spire/docs/oidc_discovery/)
+
+# Part One: Configure SPIRE Components
+
+In the first half of this procedure, you configure the SPIRE components. In the second half, you configure the AWS components.
+
+## Configuration Overview for SPIRE Components
+
+The table below summarizes the steps needed to configure the SPIRE components of OIDC federation to AWS. Follow the links for details.
+
+| Step | Action | Description |
+| --- | --- | --- |
+| Step 1 | [Prepare installation environments](#prepare-installation-environments) | Decide where you'll install SPIRE components |
+| Step 2 | [Install, configure, and run the SPIRE Server](#install-configure-and-run-the-spire-server) | Customize `server.conf`, including `ca_key_type` and `jwt_issuer` settings |
+| Step 3 | [Install, configure, and run the OIDC Discovery Provider](#install-configure-and-run-the-oidc-discovery-provider) | The SPIRE OIDC Discovery Provider serves OIDC discovery documents via a REST API |
+| Step 4 | [Install and configure SPIRE Agent](#install-and-configure-the-spire-agent) | Configure the SPIRE Agent as needed for your environment |
+| Step 5 | [Register (Attest) the Node](#register-attest-the-node) | Verify the identity of the node (machine) that the SPIRE Agent runs on |
+| Step 6 | [Register Workloads on the SPIRE Server](#register-workloads-on-the-spire-server) | Verify the identity of workloads |
+
+# Configure SPIRE for OIDC Federation
+
+Follow the procedures in the sections below to configure SPIRE for OIDC federation.
+
+## Prepare Installation Environments
+
+In a typical deployment, you would configure one machine to run the SPIRE Server and OIDC Discovery Provider and a second machine to run the SPIRE Agent. The machines can be a physical machine, virtual machine, instance, or container. The examples in these instructions do not describe how to use Kubernetes or Docker commands to set up OIDC federation, but they are supported.
+
+Review the [requirements](#required-accounts-and-permissions) described above.
+
+## Install, Configure, and Run the SPIRE Server
+
+The steps below provide an overview of installing and configuring the SPIRE Server. For more detailed information, see [Install SPIRE Server](/spire/docs/install-server/).
+
+1. Set up an instance or machine on which to run the SPIRE Server and OIDC Discovery Provider.
+
+2. [Download](/downloads/) and [install](/spire/docs/install-server/) SPIRE Server.
+
+3. Modify the following two settings in your `server.conf` SPIRE Server configuration file.
+
+| Setting | Example | Description |
+| --- | --- | --- |
+| `ca_key_type` |  `rsa-2048` | The key type used for the server certificate authority.<br/> Valid key types depend on the service to which you are configuring federation. For example, to authenticate to AWS services, the only valid key type is `rsa-2048`. AWS does not support elliptical curve keys.|
```

Comment:
A description of these two parameters would be better placed in dedicated sections in /spire/docs/configuring/.

Since this is a tutorial, instructions like this should be more prescriptive, and explain (ideally with an example) exactly how the `server.conf` file should be modified and exactly what the values should be.